spec/features/trainees/diversities/edit_disability_disclosure_spec.rb

```diff
@@ -6,18 +6,18 @@
   background do
     given_i_am_authenticated
     given_a_trainee_exists(disability_disclosure: nil)
+    given_i_visited_the_review_draft_page
+    and_i_am_on_the_disability_disclosure_page
   end
 
   scenario "choosing to disclose a disability" do
-    when_i_visit_the_disability_disclosure_page
     and_i_choose_to_disclose
     and_i_submit_the_form
     then_i_am_redirected_to_the_disabilities_page
     and_the_disability_disclosure_is_updated
   end
```

Comment:
This is becoming hard to read...

Reply:
I don't think it's too bad once you read the `background` at the top.